spec/rubocop/cop/lint/or_assignment_to_constant_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::OrAssignmentToConstant, :config do
+  subject(:cop) { described_class.new(config) }
+
+  it 'registers an offense when or-assignment to constant.' do
```

Comment:
```suggestion
  it 'registers an offense with or-assignment to a constant' do
```

Similarly for the other test descriptions as well.

Reply:
Thank you for fixing my English :sweat_smile:  
I fixed all other descriptions.